src/common/navigation.yaml

```diff
@@ -115,6 +115,7 @@ sections:
           - external:
               href: 'https://blockstack.github.io/stacks.js/'
               title: Stacks.js
+          - path: /node-configuration
```

Comment:
Just looked at the preview and I think "stacks-node-config` would be best for display in the navigation. Just "node" inside the references might not be enough for someone scanning through the navigation items